jobbergate-cli/CHANGELOG.rst

```diff
@@ -7,6 +7,9 @@ This file keeps track of all notable changes to jobbergate-cli
 Unreleased
 ----------
 
+3.2.4-alpha.0 -- 2022-08-12
+---------------------------
+
```

Comment:
maybe "Keep versions synced between the agent and the API" would be good here

Reply:
solved, thanks